ui-frontend/packages/catalog-ui-search/src/main/webapp/component/result-selector/fuzzy-results.tsx

```diff
@@ -0,0 +1,10 @@
+export const fuzzyResultCount = (resultCount: any) => {
+  if (resultCount < 10) return '< 10 hits'
```

Comment:
📝 Is it worth making this just be like the number without the word 'hits' so it's more extensible for other uses? (For instance if we want to use this in other places that don't require display or something). 

I think this is fine as is, just asking. 👍

Reply:
Good thought. I'll make that change 👍.